lms/static/js/verify_student/views/make_payment_step_view.js

```diff
@@ -59,23 +60,33 @@ var edx = edx || {};
                 this.setPaymentEnabled( true );
             }
 
+            // create a button for each payment processor
+            var buttonTemplate = _.template(
+                '<a class="next action-primary payment-button" id="<%- name %>" tab-index="0">Pay Using "<%- name %>"</a> '
+            );
+            _.each(templateContext.processors, function(processorName) {
+                html = buttonTemplate({name: processorName});
+                $( 'div.payment-buttons' ).append( html );
+            });
+
             // Handle payment submission
-            $( '#pay_button' ).on( 'click', _.bind( this.createOrder, this ) );
+            $( '.payment-button' ).on( 'click', _.bind( this.createOrder, this ) );
         },
 
         setPaymentEnabled: function( isEnabled ) {
             if ( _.isUndefined( isEnabled ) ) {
                 isEnabled = true;
             }
-            $( '#pay_button' )
+            $( '.payment-button' )
                 .toggleClass( 'is-disabled', !isEnabled )
                 .prop( 'disabled', !isEnabled )
                 .attr('aria-disabled', !isEnabled);
         },
 
-        createOrder: function() {
+        createOrder: function(event) {
```

Comment:
Seeing as how this is the "top of the chain," so to speak, this might be a good place to note that a basket will be created instead of an order if Otto is in use.